src/code/z_scene.c

```diff
@@ -232,7 +232,7 @@ void Scene_CommandSpecialFiles(PlayState* play, SceneCmd* cmd) {
     }
 
     if (cmd->specialFiles.cUpElfMsgNum != 0) {
-        play->cUpElfMsgs = Play_LoadFile(play, &sNaviMsgFiles[cmd->specialFiles.cUpElfMsgNum - 1]);
+        play->elfMessageList = Play_LoadFile(play, &sNaviMsgFiles[cmd->specialFiles.cUpElfMsgNum - 1]);
```

Comment:
I think this should keep the c-up part in some way because it's specifically for this kind of messages (see `ElfMessage_GetCUpText` vs `ElfMessage_GetSariaText`).
Also I don't know about changing it to "list" since it's very different from the actual list commands in here, and it wouldn't be consistent with the rest of the elf_message system (eg. `sChildSariaMsgs` or `gOverworldNaviMsgs`)

Reply:
See #1391